src/components/utilityFunctions.ts

```diff
@@ -32,8 +32,11 @@ export const combineRequestParams = (request: any, outputCase: OutputCase | null
 type OutputCase = 'snake' | 'camel'
 
 // Create folder if it doesn't already exist
-export const makeFolder = (folderPath: string) => {
-  if (!fs.existsSync(folderPath)) fs.mkdirSync(folderPath)
+export const makeFolder = (folderPath: string, message?: string) => {
+  if (!fs.existsSync(folderPath)) {
+    message && console.log(message)
+    fs.mkdirSync(folderPath)
```

Comment:
Just adding an optional "message" -- allows us to only print a console message if the folder is being created for the first time (won't show on subsequent runs)